src/styles/blocks/nav.scss

```
@@ -0,0 +1,49 @@
+.nav {
+&__list {
+  display: flex;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
```

check correct indents for nested styles
```suggestion
  &__list {
    display: flex;
    list-style: none;
    margin: 0;
    padding: 0;
  }
```